js/print.js

```diff
@@ -24,3 +24,18 @@ jobHistory.forEach(function(ele) {
 jobs.forEach(function(a){
   $('#jobs').append(a.toHtml());
 });
+
+// click event listeners
+handleMainNav = function() {
+  $('.icon-home').on('click', function() {
```

Comment:
There's a more dynamic way to handle this. If you match up the main nav HTML with what you want to show somehow (a data attribute, perhaps?), then you can link that up with the id of what you want to show.